### PR TITLE
Fixes for #38 and #40

### PR DESCRIPTION
**Pull request by AnneThKu**
1. Removed `#disqus thread` from post urls.
2. Added twitter meta tags.

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:description" content="UnetStack3 is a major milestone in the evolution of UnetStack, debuting a software-defined modem that can run on your laptop or on an embedded computer" />
+    <meta name="twitter:title" content="Wnat's new in UnetStack3" />
```
Comment (notthetup) [inline _includes/head.html]: Typo

File: _includes/head.html [hunk 2]
```diff
@@ -1,6 +1,12 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:description" content="UnetStack3 is a major milestone in the evolution of UnetStack, debuting a software-defined modem that can run on your laptop or on an embedded computer" />
```
Comment (notthetup) [inline _includes/head.html]: Is this going to be the content for each and every blog post?

File: _includes/head.html (hunk 1 above)
Comment (AnneThKu) [reply _includes/head.html]: Modified the title to be fetched from the post.title.

File: _includes/head.html (hunk 2 above)
Comment (AnneThKu) [reply _includes/head.html]: Removed the description as none of the posts currently have a description in the header.

File: _includes/head.html [hunk 3]
```diff
@@ -1,6 +1,11 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:title" content="{{ site.baseurl }}/{{ post.title }}" />
```
Comment (notthetup) [inline _includes/head.html]: We shouldn't need `site.baseurl` here right? We just want the `post.title`

Comment (AnneThKu) [reply _includes/head.html]: Yes, that is not required. I have removed it and pushed the fix. Also updated `twitter:image` to fetch from `post.feature-img`.

**Pull request merged**